addons/mail/data/mail_data.xml

```diff
@@ -308,7 +308,7 @@
     <span style="margin-top: 8px;">You have been assigned to the <t t-esc="model_description or 'document'"/> <t t-esc="object.display_name"/>.</span>
 </p>
 <p style="margin-top: 24px; margin-bottom: 16px;">
-    <a t-att-href="'/mail/view?model=%s&amp;res_id=%s' % (object._name, object.id)" style="background-color:#875A7B; padding: 10px; text-decoration: none; color: #fff; border-radius: 5px;">
+    <a t-att-href="link" t-att-data-oe-model="object._name" t-att-data-oe-id="object.id" style="background-color:#875A7B; padding: 10px; text-decoration: none; color: #fff; border-radius: 5px;">
```

Comment:
I would name it access_link to ease grep and understanding (link is a bit generic). Otherwise diff is small, which is great :) .